playbooks/roles/mongo_3_0/tasks/main.yml

```diff
@@ -322,7 +322,6 @@
 - name: create mongodb users in a replica set
   mongodb_user:
     database: "{{ item.database }}"
-    login_database: 'admin'
```

Comment:
This role does not work the last time i tested it. Neither does the old mongo role for that matter. (I'm talking about clustered mode, single machine mode works fine).